cmd/upgrade/main.go

```diff
@@ -0,0 +1,46 @@
+package upgrade
+
+import (
+	"fmt"
+
+	"github.com/datreeio/datree/pkg/cliClient"
+	upgrademanager "github.com/datreeio/datree/pkg/upgradeManager"
+	"github.com/spf13/cobra"
+)
+
+type Printer interface {
+	PrintMessage(messageText string, messageColor string)
+}
+type UpgradeCommandContext struct {
+	CliVersion       string
+	Printer          Printer
+	UpgradeCliClient *cliClient.CliClient
+}
+
+func New(ctx *UpgradeCommandContext) *cobra.Command {
+	m := upgrademanager.NewUpgradeManager()
+	var upgradeCmd = &cobra.Command{
+		Use:   "upgrade",
+		Short: "Upgrade datree to the latest version",
+		Long:  "Upgrade datree to the latest version",
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) > 0 {
```

Comment:
This should be checked in the "args" function, check out this example:
https://github.com/datreeio/datree/blob/4cb0e38c252a102ec8bb0b6010ebe2633dc0a19c/cmd/test/main.go#L200

Actually, even better, this check can be deleted completely! Cobra checks it by default

![Screen Shot 2022-10-23 at 22 17 28](https://user-images.githubusercontent.com/39004075/197411740-a517439a-6568-474a-b993-0fd80e7ef424.png)

Reply:
Thank you for the comment. I didn't know that 😅 . I removed the condition.